### ISSUE TITLE
docs.fivem.net/natives appears to not be updating

### ISSUE THREAD

**Issue by niekschoemaker** [citizenfx/natives#1219]
The natives on https://docs.fivem.net/natives/ don't seem to be updating for quite some time now.

Noticed this for quite some time, but thought it would be a temporary thing so waited a bit to report it.

A good example of it would be the "MUMBLE_DOES_CHANNEL_EXIST" native which has been in the game for a few months now, the documentation for this native is still not on the native reference. Same thing for some renamed natives.

Also noticed this with the downloadable json file which contains all the natives at: https://runtime.fivem.net/doc/natives.json

**Comment by spacevx** [comment]
You have to ask to @technetium-cfx

**Comment by PsychoShock** [comment]
> You have to ask to @technetium-cfx

No, @AvarianKnight is there to reports situations like that + this is already reported in intern, they are working on a "better" solution from what I had in answer. I'll alert back the people for that.

**Comment by AvarianKnight** [comment]
Going to close this as this was already fixed.
